sdk/ai/azopenai/custom_client.go

```diff
@@ -325,3 +321,12 @@ func (c *ChatRequestUserMessageContent) UnmarshalJSON(data []byte) error {
 func allowInsecure(options *ClientOptions) bool {
 	return options != nil && options.InsecureAllowCredentialWithHTTP
 }
+
+type tempAPIVersionPolicy struct{}
```

Comment:
@glecaros - we might not need this anymore. I had it in place because the /files APIs weren't setup as REST operations and weren't passing it, but that might be gone now.

Reply:
Filed as https://github.com/Azure/azure-sdk-for-go/issues/23417

Reply:
talked offline, this seems to still be needed.